src/main/java/core/basesyntax/service/impl/CsvReaderImpl.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.CsvReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.Reader;
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVRecord;
+
+public class CsvReaderImpl implements CsvReader {
```

Comment:
Reading file and parsing FruitTransaction are two different operations - do not mix them in one class
![image](https://user-images.githubusercontent.com/20326858/225299650-9a708b92-1e40-418f-9046-2be96a780b9b.png)